CHANGELOG.md

```diff
@@ -12,30 +12,62 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 > #### Migration
-> | Number of Database Clusters in Use | Requires Changes | Action Items |
-> |-----------------------------------|------------------|--------------|
-> | Single database cluster | No | No changes required |
-> | Multiple database clusters | Yes | Review all connection strings and add mandatory `clusterId` parameter ([PR #1476](https://github.com/aws/aws-advanced-jdbc-wrapper/pull/1476)). See [documentation](https://github.com/aws/aws-advanced-jdbc-wrapper/blob/main/docs/using-the-jdbc-driver/using-plugins/UsingTheFailover2Plugin.md#failover-plugin-v2-configuration-parameters) for `clusterId` parameter configuration |
+> | Number of Database Clusters in Use | Requires Changes | Action Items                                                                                                                                                                                                                                                                                                                                                                            |
+> |------------------------------------|------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+> | Single database cluster            | No               | No changes required                                                                                                                                                                                                                                                                                                                                                                     |
+> | Multiple database clusters         | Yes              | Review all connection strings and add mandatory `clusterId` parameter ([PR #1476](https://github.com/aws/aws-advanced-jdbc-wrapper/pull/1476)). See [documentation](https://github.com/aws/aws-advanced-jdbc-wrapper/blob/main/docs/using-the-jdbc-driver/using-plugins/UsingTheFailover2Plugin.md#failover-plugin-v2-configuration-parameters) for `clusterId` parameter configuration |
 
 > [!WARNING]\
 > 3.0 removes deprecated coded ([PR #1572](https://github.com/aws/aws-advanced-jdbc-wrapper/pull/1572)).
 > #### Deprecated Code Removal
 > Some methods marked as deprecated in version 2.x.x are now removed in 3.0.
```

Comment:
```suggestion
> Some methods marked as deprecated in version 2.x.x are now removed in 3.0.
> If you implemented any custom classes such as custom Plugin Service or Custom Connection Plugins, please review the changes in [PR #1572](https://github.com/aws/aws-advanced-jdbc-wrapper/pull/1572) for proper replacements.
```

Reply:
Plugin Service isn't supposed to be implemented or modified by users.